.github/workflows/test.yml

```diff
@@ -0,0 +1,23 @@
+name: test
+on:
+  pull_request
+
+jobs:
+  test:
+    runs-on: macos-12
+    name: Test
+    steps:
+      - uses: actions/checkout@v2
+      - uses: mikehardy/buildcache-action@v2
+      - name: SwiftPM Cache
+        uses: actions/cache@v3
+        with:
+          path: SourcePackages
+          key: ${{ runner.os }}-swiftpm-${{ hashFiles('**/Package.resolved') }}
+      - name: Test
+        run: |
+          xcodebuild -scheme DropcastDev -sdk iphonesimulator -destination "platform=iOS Simulator,name=iPhone 14" -resultBundlePath TestResults -skipPackagePluginValidation test
+      - uses: kishikawakatsumi/xcresulttool@v1
```

Comment:
This awesome tool shows the test results on GitHub checks.
https://github.com/kishikawakatsumi/xcresulttool

Like this.
https://github.com/konifar/Dropcast/actions/runs/4156897441/jobs/7191094518

Reply:
To work this step, you have to change GitHub Actions setting.
https://github.com/maiyama18/Dropcast/settings/actions

Settings > Actions > General > Workflow permissions
Check `Read and write permissions` and save.

Reply:
I had heard about xcresulttool but never introduced it in my repository. Looks very nice!

I have changed the settings as follows.

<img width="829" alt="image" src="https://user-images.githubusercontent.com/22269397/218346104-498dcd12-6955-468f-9edb-13f70b7975b8.png">

Reply:
Perfect! 💯 Thanks!